sdk/src/main/java/com/mapbox/maps/renderer/FpsManager.kt

```diff
@@ -0,0 +1,186 @@
+package com.mapbox.maps.renderer
+
+import android.os.Handler
+import androidx.annotation.WorkerThread
+import androidx.core.os.postDelayed
+import com.mapbox.maps.logI
+import com.mapbox.maps.logW
+import kotlin.math.pow
+
+@WorkerThread
+internal class FpsManager(
+  private val handler: Handler
+) {
+  private var userRefreshRate = USER_DEFINED_REFRESH_RATE_NOT_SET
```

Comment:
Null seems more natural for `non-set` values, but up to you

Reply:
primitive type could not be null

Reply:
Yeah but you can use Kotlin wrappers right

Reply:
I like it current way more.